torch/_inductor/scheduler.py

```diff
@@ -405,6 +416,54 @@ def can_inplace(self, read_dep: dependencies.MemoryDep):
             return read_dep.index == write_dep.index and read_dep.size == write_dep.size
         return False
 
+    def reorder_loop(
+        self, reorder: Callable[[sympy.Expr], sympy.Expr], ndim: int
+    ) -> Optional["SchedulerNode"]:
+        """Returns a new node which represents the same loop iterated in a new order
+
+        new_order represents the permutation to apply to the index variables,
```

Comment:
I don't see what new_order is?

Reply:
Ah this is out of data, it now describes the `reorder` argument.